include/ensmallen_bits/nsga2/nsga2.hpp

```diff
@@ -206,23 +206,23 @@ class NSGA2
    * @tparam MatType Type of matrix to optimize.
    * @param population The elite population.
    * @param objectives The set of objectives.
-   * @param calculatedObjectives Vector to store calculated objectives.
+   * @param calculatedObjectives Cube to store calculated objectives (numObjectives x 1 x populationSize).
```

Comment:
Sorry, I think my original suggestion didn't make sense: there is no need for a cube, a `Mat` will work just fine.  There's no need to have a second dimension that is always `1`.  Can you refactor this PR to use a `Mat` instead?